### ISSUE TITLE
Can't import the library because of wrong "main" entry in package.json

### ISSUE THREAD

**Issue by CapsE** [bluzky/nice-select2#70]
Hi, you have an error in your package.json
```
{
  "name": "nice-select2",
  "version": "2.1.0",
  "description": "A lightweight Vanilla JavaScript plugin that replaces native select elements with customizable dropdowns.",
  "main": "src/js/nice-select.js",
  "directories": {
    "doc": "docs"
  }
  ...
```
You're pointing to `src/js/nice-select.js` but the file is acutally named `nice-select2.js` this leads to issue when I try to import the library using webpack.

**Comment by imado-co** [comment]
It still exists when installed with npm

**Comment by remderk** [comment]
The problem persists when installing via npm

**Comment by jamesfawcett** [comment]
This issue is still in the latest release 2.1.0 in both this Github repo releases, and npm @CapsE

**Comment by bdube30** [comment]
Anyone know if there a way to fix this ? I use vitejs and I got this error : "Failed to resolve entry for package "nice-select2". The package may have incorrect main/module/exports specified in its package.json."

I'm pretty sure it the same kind of issue, 

I would really like to use nice-select2 for my project.

Thanks.

**Comment by Tsjippy** [comment]
fixed in version 2.2.0
